AnnService/src/Core/VectorIndex.cpp

```diff
@@ -368,6 +368,9 @@ VectorIndex::SaveIndexToFile(const std::string& p_file, IAbortOperation* p_abort
     IOBINARY(fp, WriteBinary, sizeof(configSize), (char*)&configSize);
     if ((ret = SaveIndexConfig(fp)) != ErrorCode::Success) return ret;
     configSize = fp->TellP() - sizeof(configSize);
+    if (ErrorCode::Success == ret && m_pQuantizer) {
```

Comment:
Why we need to change the load quantizer position? If you want to change, then you need to change all of the save index and load index as well as the buffersizes and filenames order you get from the index.

Reply:
Before the local-to-index quantizer change, it was needed to load the quantizer first so the right distance function would be picked. Now, it is not needed - I think it is changed in all the right places, but I can revert it.

Reply:
It is better to keep original load order : )

Reply:
returned the load order in most recent commit